murdock/job.py

```diff
@@ -61,24 +61,34 @@ def __init__(
         self.output_url = os.path.join(
             GLOBAL_CONFIG.base_url, "results", self.uid, "output"
         )
+        self._logger_context = {
+            "job": str(self.uuid),
+            "job_description": self,
```

Comment:
I would find this nicer to use `str(self)`, this way it's explicit that the `__repr__` is used (IIUC).

Reply:
You're right, added.